qctrlopencontrols/dynamic_decoupling_sequences/predefined.py

```diff
@@ -725,36 +848,69 @@ def new_quadratic_sequence(
     pre_post_rotation=False,
     **kwargs
 ):
-    """
-    Creates Quadratic Decoupling Sequence.
+    r"""
+    Creates the quadratic sequence.
 
     Parameters
     ----------
     duration : float, optional
-        defaults to None
-        The total duration of the sequence
-    number_outer_offsets : int, optional
-        Number of outer X-pi Pulses. Defaults to None. Not used if number_of_offsets
-        is supplied.
+        Defaults to None.
+        The total duration of the sequence :math:`\tau`.
     number_inner_offsets : int, optional
-        Number of inner Z-pi Pulses. Defaults to None. Not used if number_of_offsets
-        is supplied
+        Number of inner :math:`Z_{\pi}` pulses :math:`n_1`. Defaults to None.
+    number_outer_offsets : int, optional
+        Number of outer :math:`X_{\pi}` pulses :math:`n_2`. Defaults to None.
     pre_post_rotation : bool, optional
-        If True, a :math:`\\pi.2` rotation is added at the
+        If True, a :math:`X_{\pi/2}` rotation is added at the
         start and end of the sequence.
     kwargs : dict
-        Additional keywords required by
-        qctrlopencontrols.dynamical_decoupling_sequences.DynamicDecouplingSequence
+        Additional keywords required by DynamicDecouplingSequence.
 
     Returns
     -------
-    qctrlopencontrols.dynamical_decoupling_sequences.DynamicDecouplingSequence
-        Quadratic sequence
+    DynamicDecouplingSequence
+        The quadratic sequence.
 
     Raises
     ------
     ArgumentsValueError
         Raised when an argument is invalid.
+
+    See Also
+    --------
+    new_uhrig_sequence
+
+    Notes
+    -----
+    The quadratic sequence [#]_ is parameterized by duration :math:`\tau`, number of inner offsets
+    :math:`n_1`, and number of outer offsets :math:`n_2`. The outer sequence consists of
+    :math:`n_2` pulses of type :math:`X_{\pi}`, which partition the time-domain into :math:`n_2+1`
+    sub-intervals on which inner sequences consisting of :math:`n_1` pulses of type
+    :math:`Z_{\pi}`are nested. The total number of offsets is :math:`n = n_1 + n_2(n_1 + 1)`.
+
+    The pulse times for outer sequence :math:`(X_{\pi}^1, \cdots, X_{\pi}^{n_2})` are defined
+    according to the Uhrig sequence for :math:`t \in [0, \tau]`. The :math:`j`-th
+    :math:`X_{\pi}` pulse, therefore has timing offset defined by
+
+    .. math::
+        t_x^i = \tau \sin^2 \left[ \frac{i\pi}{2(n_2 + 1)}  \right] \;,
```

Comment:
The sentence after this equation says "where j=1,...,n_2" but I'm not finding a j in this formula?

Reply:
good catch. Should be `j` in the equation. Checked that in the original paper.https://arxiv.org/pdf/0908.4490.pdf